packages/terra-html-table/src/Table.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Table.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Whether or not the rows should be zebra striped
+   */
+  isStriped: PropTypes.bool,
+  /**
+   * Whether or not the table cells should be padded
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+};
+
+const defaultProps = {
+  isStriped: true,
+  paddingStyle: 'none',
+};
+
+const Table = ({
+  children,
+  isStriped,
+  paddingStyle,
+  ...customProps
+}) => {
+  const tableClassNames = cx([
+    'table',
+    { striped: isStriped },
+    { 'padding-standard': paddingStyle === 'standard' },
+    { 'padding-compact': paddingStyle === 'compact' },
+    customProps.className,
```

Comment:
I think @mjhenkes pointed this out in another PR, we should honor custom classnames, but we shouldn't be binding them to our style sheet. I think this is something we've been doing in all components that needs fixed

Reply:
[cx(...), customProps.className].join(' ').trim()